### PR TITLE
feat: add CI to check yarn build and yarn format

### PR DESCRIPTION
**Pull request by firestack**
This tripped me up in #5

### CODE REVIEW

**Comment by firestack** [comment]
[Echoing my comment on the other PR](https://github.com/mbta/github-asana-action/pull/8#issuecomment-2091138530)

Let me know if you think the package version should be bumped for this?

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+on: [push, pull_request]
+
+jobs:
+  assert-formatted:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```
Comment (paulswartz) [inline .github/workflows/check.yml]: question: do we care about the specific version?
```suggestion
          node-version: lts
```

Comment (firestack) [reply .github/workflows/check.yml]: I was matching with what's in `tool-versions` but I'm fine with this.

Comment (paulswartz) [reply .github/workflows/check.yml]: Oh I didn't realize it was there: looks like you can use `node-version-file` to get it from there: https://github.com/actions/setup-node/blob/main/action.yml#L10-L11

**Review by paulswartz — APPROVED** [review]
suggestion(non-blocking): up to you whether you want to look at the `node-version-file` option

**Pull request merged**